Source/Infrastructure/Integration/Kysect.Shreks.Integration.Google/Tools/ISheetManagementService.cs

```diff
@@ -1,10 +1,13 @@
-﻿using Kysect.Shreks.Integration.Google.Sheets;
-
-namespace Kysect.Shreks.Integration.Google.Tools;
+﻿namespace Kysect.Shreks.Integration.Google.Tools;
 
 public interface ISheetManagementService
 {
-    Task CreateOrClearSheetAsync(string spreadsheetId, ISheet sheet, CancellationToken token);
+    /// <summary>
+    /// Clears all values and formatting of a sheet
+    /// if it exists and creates it otherwise
+    /// </summary>
+    /// <returns>Sheet id</returns>
+    Task<int> CreateOrClearSheetAsync(string spreadsheetId, string sheetTitle, CancellationToken token);
 
     Task<string> CreateSpreadsheetAsync(string title, CancellationToken token);
```

Comment:
Для CreateSpreadsheetAsync тоже добавь.

```suggestion
    /// <summary>
    ///     Clears all values and formatting of a sheet
    ///     if it exists and creates it otherwise
    /// </summary>
    /// <returns>Sheet id</returns>
    Task<int> CreateOrClearSheetAsync(string spreadsheetId, string sheetTitle, CancellationToken token);

    Task<string> CreateSpreadsheetAsync(string title, CancellationToken token);
```